### PR TITLE
Survey / Anna-Kaisa Honkonen

### PR DESCRIPTION
**Pull request by honkoan**
Link to my deployed project:
https://peaceful-kepler-96f7b6.netlify.app/

My reflections on how this week's project turned out:
I liked this week’s project. I did it for my own company so I can actually use the site in the future after some changes (e.g. make the site to save user responses). I didn't have much time this week so I did this project quite fast and I'm proud of what I managed to do. 

Things I'd like to have clarified or explained in more detail:
I can't point out any specific things that I'd like to have clarified. The lectures were really good and Maks and Van explained things very well. I need some time to understand everything but the teachers are super good (and patient)! :)

### CODE REVIEW

File: code/src/components/Week.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+          <label htmlFor="week">Select a week:</label>
+          <input
+            type="week"
+            id="week"
+            name="week"
+            weekvalue={whichWeek}
+            onChange={onWhichWeekChange}
+          />
+        </div>
+      </form>
```
Comment (dannuzak) [inline code/src/components/Week.js:18]: You added a calendar! that's fancy, well done! 👯

File: code/src/index.css [hunk 2]
```diff
@@ -1,13 +1,55 @@
...
+
+button:hover {
+  background-color: #fb0000
+}
+
+button:active {
+  background-color: #e94545;
+  box-shadow: 0 1px black;
+  transform: translateY(6px);
+}
```
Comment (dannuzak) [inline code/src/index.css:55]: Nice and short! Great! 👍🏼

**Review by dannuzak — COMMENTED** [review]
Hi Anna-Kajsa, It was a pleasure to do a code review for your project. i didn't find much advice to give or corrections to suggest tho... cause it looks great to me!

**Review by KarolCislowski — APPROVED** [review]
Great job! 
For the next step, it would be nice to look for info on how to validate users' inputs :)

**Pull request closed without merging**